editor/animation_track_editor.cpp

```diff
@@ -3232,7 +3232,9 @@ void AnimationTrackEditGroup::_notification(int p_what) {
 			int ofs = 0;
 			draw_texture(icon, Point2(ofs, int(get_size().height - icon->get_height()) / 2));
 			ofs += separation + icon->get_width();
+
 			draw_string(font, Point2(ofs, int(get_size().height - font->get_height(font_size)) / 2 + font->get_ascent(font_size)), node_name, HORIZONTAL_ALIGNMENT_LEFT, timeline->get_name_limit() - ofs, font_size, color);
+			node_name_rect = Rect2(0, 0, timeline->get_name_limit(), get_size().height);
```

Comment:
It doesn't look like it needs to be assigned here. The variable uses only object's fields, so it can be made local to `gui_input()`.